rds/main.tf

```diff
@@ -1,12 +1,14 @@
 locals {
   engine_nickname = "${var.engine == "postgres" ? "pg" : "mysql"}"
+  option_group = "${var.engine == "mysql" ? true : false}"
```

Comment:
I think we should compare to postgres here and flip the boolean to match what we do in the engine_nickname

Reply:
should we just call this `is_mysql`